src/DependencyInjection/StofDoctrineExtensionsExtension.php

```diff
@@ -44,6 +44,7 @@ class StofDoctrineExtensionsExtension extends Extension
         'softdeleteable' => array(
             'loadClassMetadata',
             'onFlush',
+            'postFlush',
```

Comment:
this is now already fixed in the `main` branch (as it is separate from the new IP address provider). It would be great to rebase your branch to clean this diff.